lib/rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Kindly ensure that `Getters` and `Setters` for `:book` and `:person` are present. Currently, you only have `getters` set for `:book` and `:person` so kindly fix that :wink: